server/routes/treeadoptions/treeadoptions-queries.js

```diff
@@ -9,7 +9,9 @@ export async function findTreeAdoptionsByTreeId(id) {
     WHERE id = $1;
   `;
   const values = [id];
-  const treeAdoptions = db.manyOrNone(query, values);
+  const treeAdoptions = db.manyOrNone(query, values)
+    .then(data => data)
+    .catch(error => error);
```

Comment:
I think this is swallowing any errors, preventing them from being handled anywhere else.  Shouldn't they be logged, at least?

Reply:
That's very strange.  Can't really see how it could have any effect, since it's just returning exactly what it received to the next handler in the chain.